analytics.py

```diff
@@ -0,0 +1,274 @@
+#analytics
+import math
+import json
+import sys 
+import os
+
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
In a normal package, that was being installed by another user, you would not need this line for path modification.  This is `nose` behaving weirdly (unfortunately).